src/analysis/retail/hunter/survival/modules/talents/WildfireBomb.tsx

```diff
@@ -194,6 +259,37 @@ class WildfireBomb extends Analyzer {
     );
   }
 
+  get guideSubsection(): JSX.Element {
+    const explanation = (
+      <p>
+        <strong>
+          <SpellLink spell={TALENTS.WILDFIRE_BOMB_TALENT} />
+        </strong>{' '}
+        should be kept off maximum charges and always be cast with{' '}
+        <SpellLink spell={SPELLS.TIP_OF_THE_SPEAR_CAST.id} />. It can go untipped if any of:
+        <br></br>
+        1. You are capped on bomb charges. <br></br>
+        2. Lunar Storm is ready <br></br>
+        3. You are about to press Coordinated Assault and have{' '}
+        <SpellLink spell={TALENTS.BOMBARDIER_TALENT} /> talented.<br></br>
+        4. You are about to press Butchery and the cooldown reduction from Frenzied Strikes would
+        overcap bomb.
+      </p>
```

Comment:
don't use `<br/>` for line formatting. it breaks on differing screen sizes. for this case, you can use `<ol>` and `<li>` to get a formatted ordered list with proper spacing and automatic line breaks.